mobius-test/src/test/java/com/spotify/mobius/test/InitSpecTest.java

```diff
@@ -60,4 +61,14 @@ public void initTests() throws Exception {
         .when("bad model")
         .thenError(error -> assertThat(error, instanceOf(IllegalStateException.class)));
   }
+
+  @Test
+  public void shouldFailIfExpectedErrorDoesntHappen() throws Exception {
+    assertThatThrownBy(
+            () ->
+                initSpec
+                    .when("no crash here")
+                    .thenError(error -> assertThat(error, instanceOf(IllegalStateException.class))))
+        .isInstanceOf(AssertionError.class);
```

Comment:
Is there a reason this test doesn't check the message like the one for UpdateSpec?